build.sbt

```diff
@@ -318,10 +318,18 @@ lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
         <url>http://github.com/fwbrasil/</url>
       </developer>
     </developers>)
-) ++ update2_12
+) ++ `scala 2.12 hacks`
 
-lazy val update2_12 = Seq(
-  scalacOptions -= (
-    if (scalaVersion.value.startsWith("2.12.")) "-Xfatal-warnings" else ""
-  )
-)
+lazy val `scala 2.12 hacks` = Seq(
```

Comment:
this one brings back `-Xfatal-warnings` for 2.12, but fails only for unused imports rather than all others (params, patvars and so on). So with this new hack we bring the behavior as with 2.11